scrapy/core/http2/protocol.py

```diff
@@ -33,13 +33,17 @@
 logger = logging.getLogger(__name__)
 
 
+ACCEPTABLE_PROTOCOL = b"h2"
```

Comment:
:lipstick: Maybe simply `PROTOCOL`? I’m definitely [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality) here, so feel free to mark this as solved without any change.

Reply:
I don't much like `ACCEPTABLE_PROTOCOL` to be honest, but I think I'd prefer to go with `PROTOCOL_NAME`, for extra explicitness.